content/altinn-studio/guides/development/signing/role-based-signing/backend-manual/add-process-task.en.md

```
@@ -38,9 +49,9 @@ Signing normally uses two user actions. If the Altinn user interface is used by
 </bpmn:task>
 ```
 
+If you have chosen to send signing receipts by specifying a correspondence resource, you can find additional details that configuration [here](/altinn-studio/guides/development/signing/runtime-delegated-signing/#configure-environment-specific-correspondence-resources).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)